gendoc.go

```diff
@@ -7,7 +7,7 @@ package cityaq
 //    and put in $PATH
 
 // Install the code generation dependencies.
-//go:generate bash -c "rm ~/go/bin/protoc-gen-go"
+//go:generate bash -c "rm -rf ~/go/bin/protoc-gen-go"
```

Comment:
This is deleting a single file, right? If so, why would it need to be recursive?

Reply:
yeah right. You can drop the from it. Both work for me though still.